src/PostSharp.Engineering.BuildTools/Utilities/ToolInvocationHelper.cs

```diff
@@ -195,7 +197,8 @@ public static bool InvokeTool(
             out int exitCode,
             Action<string> handleErrorData,
             Action<string> handleOutputData,
-            ToolInvocationOptions? options = null )
+            bool isOutputImportant,
+            ToolInvocationOptions? options )
```

Comment:
Expose the `isOutputImportant` into ToolInvocationOption as `InputReadingTimeout` with a default value of 10.